yt_dlp/extractor/fujitv.py

```diff
@@ -29,8 +30,11 @@ class FujiTVFODPlus7IE(InfoExtractor):
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
+        self._download_webpage(url, video_id, fatal=False)
         formats = self._extract_m3u8_formats(
             self._BASE_URL + 'abr/tv_android/%s.m3u8' % video_id, video_id, 'mp4')
+        token = self._get_cookies(url).get('CT').value
+        json_info = self._download_json('https://fod-sp.fujitv.co.jp/apps/api/episode/detail/?ep_id=%s&is_premium=false' % video_id, video_id, headers={'x-authorization': f'Bearer {token}'})
```

Comment:
if the request can't happen without the token, then add a check for it before making request and if it's not present then raise error.

Reply:
ok

Reply:
should that be `ExtractorError`?

Reply:
The video can be downloaded without this request succeeding. So this shouldn't throw error and should be non-fatal

Reply:
so need to provide a fallback for the `title`.

Reply:
the `_download_webpage` is for getting the token, the request which needs the token is `_download_json`, so I will add the `fatal=False` to that.